src/wp-includes/block-patterns.php

```diff
@@ -243,6 +267,7 @@ function _load_remote_featured_patterns() {
  * `theme.json` file.
  *
  * @since 6.0.0
+ * @since 6.2.0 Normalize the pattern from the API (snake_case) to the format expected by `register_block_pattern` (camelCase).
```

Comment:
```suggestion
 * @since 6.2.0 Normalized the pattern from the API (snake_case) to the
  *                       format expected by `register_block_pattern()` (camelCase).
```

Line length for readability.